java/vector/src/test/java/org/apache/arrow/vector/ipc/MessageSerializerTest.java

```diff
@@ -148,7 +150,7 @@ public void testSchemaDictionaryMessageSerialization() throws IOException {
   public ExpectedException expectedEx = ExpectedException.none();
 
   @Test
-  public void testSerializeRecordBatch() throws IOException {
+  public void testSerializeRecordBatchV4() throws IOException {
```

Comment:
doesn't this need an `IpcOption` to set `MetadataVersion.V4`?

Reply:
Good catch, thank you. I've added the IpcOption.